diff --git a/src/Kernel.php b/src/Kernel.php
index 1cd0572..83eb771 100644
--- a/src/Kernel.php
+++ b/src/Kernel.php
@@ -51,4 +51,14 @@ class Kernel extends BaseKernel
         $routes->import($confDir.'/{routes}/*'.self::CONFIG_EXTS, '/', 'glob');
         $routes->import($confDir.'/{routes}'.self::CONFIG_EXTS, '/', 'glob');
     }
+
+    public function getLogDir()
+    {
+        return '/tmp/sf/logs';
+    }
+
+    public function getCacheDir()
+    {
+        return '/tmp/sf/cache';
+    }
 }
